experiment/slack-oncall-updater/main.go

```diff
@@ -26,6 +26,31 @@ import (
 	"strings"
 )
 
+// githubToSlack maps (lowercase!) GitHub usernames to Slack user IDs.
+// Add yourself when you join the oncall rotation.
+// You can mostly easily find your user ID by visiting your Slack profile, clicking "...",
+// and clicking "Copy user ID".
+var githubToSlack = map[string]string{
+	"amwat":          "U9B1P2UGP",
+	"chases2":        "UJ9R0FWD6",
+	"cjwagner":       "U4QFZFMCM",
+	"bentheelder":    "U1P7T516X",
+	"fejta":          "U0E2KHQ13",
+	"katharine":      "UBTBNJ6GL",
+	"michelle192837": "U3TRY5WV7",
+	"mushuee":        "U013TPFJWC8",
+	"spiffxp":        "U09R2FL93",
+}
+
+// rotationToSlack group maps the rotation in go.k8s.io/oncall to the slack
+// group ID for the oncall alias group for this rotation
+var rotationToSlackGroup = map[string]string{
+	"test-infra":         "SGLF0GUQH",   // @test-infra-oncall
+	"google-build-admin": "S017N31TLNN", // @google-build-admin
+	// TODO: create this slack group?
+	// "scalability": "",
```

Comment:
FYI @wojtek-t, this tool enables go.k8s.io/oncall data to map to a slack group so instead of pinging random specific people the release team etc. can ping `@test-infra-oncall` => the current test-infra-oncall

You could enable this for scalability oncall by:
- have each member of the rotation add themselves to `githubToSlack` above, following the instructions above
- sending a PR like https://github.com/kubernetes/community/pull/4989 to add an external-managed group with some name to the slack config
- asking #slack-admins to manually create the group with this name
- `@mention` the group from the slack web client, alt/right click & inspect the name. Grab the `data-user-group-id` property value, add it to this mapping on this line.